app/controllers/application_controller.rb

```diff
@@ -2,6 +2,6 @@ class ApplicationController < ActionController::Base
   helper_method :current_user
 
   def current_user
-    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
These two lines are doing the same thing. The `find` method automatically looks up an DB entry by the id passed into it. Curious why this change was made.